pages/product/[name].js

```diff
@@ -30,7 +29,7 @@ const Product = ({ data }) => {
           </div>
           <HeartIcon className="absolute top-0 right-0 h-[50px] w-[50px] bg-white p-2" />
           <div className="relative flex h-[600px] w-full flex-col items-center">
-            <NextImage layout="fill" objectFit="contain" src={data.images[selectedImage].url} className="h-auto w-full max-w-[600px]" />
+            <img src={relativizeURL(data.images[selectedImage].url)} className="h-auto w-full max-w-[600px] object-contain" />
```

Comment:
IMages are brutally slow. Can we add Image Optimization?

Reply:
Oh yes, Added in https://github.com/layer0-docs/layer0-nextjs-example/pull/117/commits/b0ba9e472c778434b501d7153299b55ccd7b485b